README.md

```diff
@@ -8,19 +8,35 @@ For a more comprehensive example of Wix Headless integration, we recommend check
 our [starter templates](https://www.wix.com/developers/headless/templates).
 
 You can view the latest version of this repo deployed
-at [https://wix-headless-example.vercel.app/](https://wix-headless-example.vercel.app/), but we recommend cloning it and
-experimenting with it locally.
+at [https://wix-headless-example.vercel.app/](https://wix-headless-example.vercel.app/).
```

Comment:
Maybe add here that the Vercel deployment uses an example backend project with sample data.